src/Components/PresentationalComponents/WithLoader/WithLoader.test.js

```diff
@@ -1,26 +1,66 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import toJson from 'enzyme-to-json';
 import WithLoader from './WithLoader';
+import { render } from '@testing-library/react';
+import { CVEPageContext } from '../../SmartComponents/CVEDetailsPage/CVEDetailsPage';
 
 describe('WithLoader component', () => {
-    it('should render without props', () => {
-        const wrapper = shallow(<WithLoader />);
-        expect(toJson(wrapper)).toMatchSnapshot();
-    });
-
     it('should render with loading prop is undefined', () => {
-        const wrapper = shallow(<WithLoader isLoading={undefined} />);
-        expect(toJson(wrapper)).toMatchSnapshot();
+        const { asFragment } = render(
+            <WithLoader isLoading={undefined}>
+                children
+            </WithLoader>
+        );
+        expect(asFragment()).toMatchSnapshot();
```

Comment:
Looking at the snapshots of these tests here they don't really tell anything, they didn't before either. Maybe we can either looking into making these render properly and query something specific(ed in the test title) or we remove them. The latter should really be a last resort.

Reply:
They do tell something. Here's a few examples:

- https://github.com/RedHatInsights/vulnerability-ui/pull/2069/files#diff-03a14f7e9ded9af81700d4f7526431abce962da6c1c80e49f8885dfdbcdeff9bR54
- https://github.com/RedHatInsights/vulnerability-ui/pull/2069/files#diff-03a14f7e9ded9af81700d4f7526431abce962da6c1c80e49f8885dfdbcdeff9bR91
- https://github.com/RedHatInsights/vulnerability-ui/pull/2069/files#diff-03a14f7e9ded9af81700d4f7526431abce962da6c1c80e49f8885dfdbcdeff9bR103
- ```
exports[`WithLoader component should render with CompactTable 1`] = `
<DocumentFragment>
  <table
    aria-label="Loading"
    class="pf-c-table pf-m-grid-md pf-m-compact"
```
- ```
exports[`WithLoader component should render with SkeletonTable 1`] = `
<DocumentFragment>
  <table
    aria-label="Loading"
    class="pf-c-table pf-m-grid-md"
```
- This renders children when loading is false:
```
exports[`WithLoader component should render with loading prop is false 1`] = `
<DocumentFragment>
  children
</DocumentFragment>
`;
```
- This renders children when loading is undefined:
```
exports[`WithLoader component should render with loading prop is undefined 1`] = `
<DocumentFragment>
  children
</DocumentFragment>
`;
```

Reply:
@johnsonm32 Oh, my bad. You are right they do tell something. 

I still would argue that they might be better querying something specific and make it more obvious what the test is looking for and what the actual difference should be. For example, we could query the "Loading" label for the loading state. Or check for the "children" text when it should not render the loading state. But we can do that at a later point.